aeron-cluster/src/main/java/io/aeron/cluster/ClusterBackup.java

```diff
@@ -251,6 +251,16 @@ public void close()
          */
         public static final String CLUSTER_BACKUP_PROGRESS_TIMEOUT_PROP_NAME = "aeron.cluster.backup.progress.timeout";
 
+        /**
+         * Interval at which the cluster backup is re-initialised after an exception has been thrown.
+         */
+        public static final String CLUSTER_BACKUP_COOLOFF_INTERVAL_PROP_NAME = "aeron.cluster.backup.cooloff.interval";
```

Comment:
COOLDOWN is already used in Aeron so would be a better fit.

Reply:
👍 will rename.